buildenv/jenkins/common/pipeline-functions

```diff
@@ -685,4 +687,13 @@ def setup_pull_request() {
     PERSONAL_BUILD = 'true'
 }
 
+def strip_cmake_specs(spec) {
+    // Strip off cmake from SPEC name
+    // Since cmake is a method for how the SDK is built,
+    // from a test perspective it is the same sdk. Therefore
+    // we will reuse the same test jobs as the non-cmake specs.
+    spec -= '_cm'
+    echo "Strip cmake spec:${spec} -> ${new_spec}"
```

Comment:
Where is the `new_spec ` defined?

Reply:
Good catch, I had changed it slightly after the test build. I will re-test.

Reply:
Fixed @llxia